install-dev/upgrade/upgrade.php

```diff
@@ -34,6 +34,60 @@
 $engineType = 'ENGINE_TYPE';
 define('PS_IN_UPGRADE', 1);
 
+$incompatibleModules = [
```

Comment:
I'm not sure that this array should be here…

Right now it might works, but only for 1.6 to 1.7. If for the 1.8 we re-add a module named … blockcms for example, this module will be on the ignored list.

This script, I think, is called only one time when you upgrade.